templates/endpoints_v2/README.md

```diff
@@ -0,0 +1,115 @@
+# Endpoints - Deploy, configure, and serve LLMs 
+
+The guide below walks you through the steps required for deployment of LLM endpoints. Based on Ray Serve and RayLLM, the foundation for [Anyscale-Hosted Endpoints](http://anyscale.com/endpoints), the Endpoints template provides an easy to configure solution for ML Platform teams, Infrastructure engineers, and Developers who want to deploy optimized LLMs in production.  We have provided a number of examples for popular open-source models (Llama2, Mistral, Mixtral, embedding models, and more) with different GPU accelerator and tensor-parallelism configurations in the `models` directory. 
+
+# Step 1 - Deploy the model on Workspace
+
+The serve.yaml file in this example runs the Mistral-7B model. There are 2 important configurations you would need to modify:
+1. The `models` config in `serve.yaml` contains a list of YAML files for the models you want to deploy. You can any of the models in the `models` directory or define your own model YAML file and run that instead. Follow the CustomModels [guide](CustomModels.md) for bringing your own models.
+2. `HUGGING_FACE_HUB_TOKEN` - The Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN environment variable to be set to a Hugging Face Access Token for an account with permissions to download the model.
+
+From the terminal use the Ray Serve CLI to deploy a model:
+
+```shell
+# Deploy the Mistral-7b model. 
+
+serve run serve.yaml
+```
+
+# Step 2 - Query the model
+
+Once deployed you can use the OpenAI SDK to interact with the models, ensuring an easy integration for your applications. Run the following command in a separate terminal to query. 
+
+```shell
+python query.py
+```
+```text
+Output:
+The top rated restaurants in San Francisco include:
+ • Chez Panisse
+ • Momofuku Noodle Bar
+ • Nopa
+ • Saison
+ • Mission Chinese Food
+ • Sushi Nakazawa
+ • The French Laundry
+```
+
+Endpoints uses an OpenAI-compatible API, allowing us to use the OpenAI SDK to access Endpoint backends.
+
+```python
+from openai import OpenAI
+
+client = OpenAI(
+  base_url="http://localhost:8000/v1",
+  api_key="NOT A REAL KEY",
```

Comment:
I have noticed customers not knowing how to get auth key when they run `anyscale service rollout`

Reply:
I'll add a comment here